test/puppetlabs/puppetdb/config_test.clj

```diff
@@ -173,6 +169,15 @@
                      (.mkdir)
                      (.setWritable true))]
       (is (= (validate-vardir (vardir filename))
+             (vardir filename)))))
+
+  (testing "should support relative paths"
+    (let [filename "./target/totally/okay"]
+      (doto (fs/file filename)
+        (.deleteOnExit)
+        (.mkdirs)
```

Comment:
You cold probably use our testutils function for doing some of this https://github.com/puppetlabs/puppetdb/blob/master/test/puppetlabs/puppetdb/testutils.clj#L254